examples/full-example/.mendelrc_v2

```diff
@@ -34,6 +34,8 @@ transforms: # a list of all available transforms for all envs and types
         - babili
   json-parse:
     plugin: mendel-parser-json
+  atomic-css:
+    plugin: mendel-atomic-css
```

Comment:
Well, I would prefer to not mix this into Mendel. It is kind of a line between proprietary and open source here.